participants/thaosle.sql

```diff
@@ -1,26 +1,38 @@
 -- Question 1
--- The youngest pet is Anna. (birthday is 2024-10-12)
-    -- Commands:
-        -- SELECT *
-        -- FROM pet
-        -- ORDER BY dateOfBirth DESC
-        -- LIMIT 1
+/* The youngest pet is Anna. (birthday is 2024-10-12)
+    Commands:
+        SELECT *
+        FROM pet
+        ORDER BY dateOfBirth DESC
+        LIMIT 1
+*/
 -- Question 2
--- Yes, you would just have to query the records that contains a conditional
--- that would have filter out any birthdays that happened 3-4 years ago
---  (there should be 2 records)
-    -- Commands:
-        -- SELECT *
-        -- FROM pet
-        -- WHERE YEAR(CURRENT_DATE()) - YEAR(dateOfBirth) = 3 OR YEAR(CURRENT_DATE() - YEAR(dateOfBirth) = 4
+/* Yes, you would just have to query the records that contains a conditional
+   that would have filter out any birthdays that happened 3-4 years ago
+   (there should be 2 records)
+    Commands:
+        SELECT *
+        FROM pet
+        WHERE YEAR(CURRENT_DATE()) - YEAR(dateOfBirth) = 3 OR YEAR(CURRENT_DATE() - YEAR(dateOfBirth) = 4
```

Comment:
Let's assume a pet was born on the 22nd of November 2021 and today is October 14th, 2024, making our pet 2 years old at the moment. However, `Select YEAR(CURRENT_DATE())-YEAR("2021-11-22") as 'age';` would yield 3. 
This approach unfortunately doesn't work.